rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -973,6 +974,36 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> guiSettings, std:
 
 	xemu_settings->EnhanceCheckBox(ui->spuDebug, emu_settings::SPUDebug);
 	SubscribeTooltip(ui->spuDebug, json_debug["spuDebug"].toString());
+		
+	if (utils::has_rtm() && !utils::has_mpx())
+	{
+		xemu_settings->EnhanceCheckBox(ui->enableTSX, emu_settings::EnableTSX);
```

Comment:
In order to make this universal change this to:
xemu_settings->EnhanceComboBox(ui->enableTSX, emu_settings::EnableTSX);